index.html

```diff
@@ -34,5 +33,187 @@ <h1 class="header" >I’m Mariane<br>
+                </nav> 
+        </section>
+
+        <section class="container_resume" >
+            <article class="resume_border">
+                <h2 class="heading_resume">About<br>Myself</h2>    
+            <p class="content_resume">Hello I’m a software developer! I can help you build a product , feature or website
+                     Look through some of my work and experience! If you like what you see and have a 
+                     project you need coded, don’t hestiate to contact me.</p>
+                     <a class="connect_resume" href="#"><img src="image/Social Media.svg" alt="Let's Connect"></a>
+                     <div class="social_resume">
+                        <ul>
+                            <li><a href="#"><img class="github" src="image/github.svg" alt="GitHub"></a></li>
+                            <li><a href="#"><img class="linkedin" src="image/Linkedin icon.svg" alt="Linkedin"></a></li>
+                            <li><a href="#"><img class="angellist" src="image/angellist.svg" alt="Angellist"></a></li>
+                            <li><a href="#"><img class="twitter" src="image/twitter.svg" alt="twitter"></a></li>
+                            <li><a href="#"><img class="medium" src="image/medium.svg" alt="Medium"></a></li>
+                            
+                            
+                        </ul>
+                     </div>
+                     <nav class="nav-button-resume">
+                        <a href=""><input type="button" class="resume-button" value="Get my resume"></a> 
+                    </nav>
+            </article>
+            <section class="border_skills">
+                <article class="border_language">
+                <div class="language_button">
+                    <h3 class="languages">Languages</h3>
+                    <img class="lang_button" src="image/language_button.svg" alt="Lunguage button">
+                </div>
+            <div class="language">
+                <ul>
+                    <li><img src="image/Frame 9.svg" alt="Javascript"></li>
+                    <li><img src="image/Frame 10.svg" alt="Javascript"></li>
+                    <li><img src="image/Frame 11.svg" alt="Javascript"></li>
+                </ul>
+            </div>
+                </article>
+                <div class="flex_bar">
+                    <h3 class="flex_frame">Frameworks</h3>
+                    <img class="chevron" src="image/Union.svg" alt="icon-chevron">
+                </div>
+                    <img  class="separator" src="image/Separator Bottom.svg" alt="separator">
+                </div>
+                <div class="flex_bars">
+                        <h3 class="flex_skill">Skills</h3>
+                        <img class="chevron" src="image/Union.svg" alt="icon-chevron">
+                </div>
+                    <img class="separator" src="image/Separator Bottom.svg" alt="separator">
+                </div>
+        </section>
```

Comment:
- [ ] `About Myself` is a section that comprises of this part. It is not semantically ok to use `section` tag inside here again. I would recommend instead to use generic `div` in place of the `section` tag. Kindly change the `section` tag to `div` tag.